### DISCUSSION TITLE
Loadouts feel... bland.

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#6683] [Unstable feedback]
The renaming of all the loadouts to loadout 1, 2 and 3 feels extremely boring, The removal of the descriptions also makes them feel less fleshed out, and having no name or description makes it more difficult for new players to figure out which loadout suits them.

**Comment by Rokvach** [comment]
That was changed because it didn't make much sense to have them when we have the new talent system which also has sub-classes
